index.html

```diff
@@ -18,7 +18,7 @@ <h2 class="project-tagline">A responsive theme for your project on GitHub Pages<
     </section>
 
     <section class="main-content">
-      <p>Text can be <strong>bold</strong>, <em>italic</em>, or <del>strikethrough</del>. <a href="http://github.com">Links</a> should be blue with no underlines (unless hovered over).</p>
+      <p>Text can be <strong>bold</strong>, <em>italic</em>, or <del>strikethrough</del>. <a href="https://github.com">Links</a> should be blue with no underlines (unless hovered over).</p>
```

Comment:
Yep - this was just reflex, but harmless. github.com is HSTS so it's likely the user wouldn't go through a redirect anyway.